src/dlgTriggerEditor.cpp

```diff
@@ -345,7 +345,7 @@ dlgTriggerEditor::dlgTriggerEditor( Host * pH )
     connect( showSearchAreaAction, SIGNAL(triggered()), this, SLOT( slot_show_search_area()));
 
     QAction * saveAction = new QAction( QIcon( QStringLiteral( ":/icons/document-save-as.png" ) ), tr("Save Item"), this);
-    //saveAction->setShortcut(tr("Ctrl+S"));
+    saveAction->setShortcut(tr("Ctrl+S"));
```

Comment:
Well, the problem I guess he would be mentioning would be that having Ctrl+s getting captured by the window as a shortcut prevents you from setting up a new Macro using that key combo. It does not prevent you from running any existing Macros with that key combo however. Not sure how that could be fixed, ultimately.

As far as the shortcut keys go, I guess alt+s would work as well, although it seems a little counter-intuitive to me since I've always thought of ctrl+s as more of saving your immediate work, rather than saving an entire project. I don't think SaveAs seems like something that should be getting enough use to even warrant a shortcut key, although eventually I'd like to see Mudlet set up so that navigating it is very easily done without the use of a mouse at all, which would require some additional changes that I'm pondering.